prime-router/src/test/kotlin/metadata/MapperTests.kt

```diff
@@ -846,6 +846,37 @@ class MapperTests {
         assertThat(mapper.apply(lookupElement, args, elementAndValuesUNK).value).isNull()
     }
 
+    @Test
+    fun `test LookupSenderAutomationValuesetsMapper`() {
+        val table = LookupTable
+            .read("./src/test/resources/metadata/tables/sender_automation_value_set_row.csv")
+        val schema = Schema(
+            "test",
+            topic = "covid-19",
+            elements = listOf(
+                Element(
+                    "gender",
+                    type = Element.Type.TABLE,
+                    table = "sender_automation_value_set_row",
+                    tableColumn = "code",
+                    mapperOverridesValue = true
+                )
+            )
+        )
+        val metadata = Metadata(
+            schema = schema,
+            table = table,
+            tableName = "sender_automation_value_set_row"
+        )
+
+        val lookupElement = metadata.findSchema("test")?.findElement("gender") ?: fail("")
+        val mapper = LookupSenderAutomationValuesetsMapper()
+        val args = listOf("gender")
+        val elementAndValues = listOf(ElementAndValue(lookupElement, "Female"))
+        assertThat(mapper.valueNames(lookupElement, args)).isEqualTo(listOf("gender"))
+        assertThat(mapper.apply(lookupElement, args, elementAndValues).value).isEqualTo("F")
```

Comment:
We should make a test for when it can't find a value to map so we have failure coverage as well